src/goCover.ts

```diff
@@ -140,6 +148,13 @@ export function getCoverage(coverProfilePath: string, showErrOutput: boolean = f
 				let fileRange = data.match(/([^:]+)\:([\d]+)\.([\d]+)\,([\d]+)\.([\d]+)\s([\d]+)\s([\d]+)/);
 				if (!fileRange) return;
 
+				// I really don't know how to fix this elsewhere. I get an underscores in front o thef path (on macOS)
+				//  in go-code-cover file. This fixes it.
+				// _/Users/me/.../file.go => /Users/me/.../file.go
+				if (fileRange[1].charAt(0) === '_') {
```

Comment:
You might not have GOPATH set. Can you run the command `Go: Current GOPATH` and see what is the result?

See https://stackoverflow.com/questions/31664129/executing-go-tests-underscore-in-place-of-correct-path